www/md_base/src/app/common/common.constant.coffee

```diff
@@ -5,17 +5,27 @@ invert_constant = (constant_name, inverted_constant_name) ->
 
 class Baseurlapi extends Constant('common')
     constructor: ->
-        return 'api/v2/'
```

Comment:
@tardyp The biggest challenge is that we cannot get the `configjson` without python backend.